tests/scripts/other_langs/r_package_streamlined_fail/run_model.R

```diff
@@ -0,0 +1,68 @@
+library(jsonlite)
```

Comment:
Just a thought, but is it necessary to keep two versions of this run_model script (for the streamlined and streamlined_fail runs)? I see that the test fixtures get set up by each subdirectory. but it might be good for adding future tests as well if the same model/wrapper can be run with multiple config files

Reply:
Yeah, I can change it so that it has 2 config files in the streamlined folder and then just change the test.